server_admin/topics/clients/client-oidc.adoc

```diff
@@ -149,6 +149,21 @@ WARNING:
 None of the keycloak client adapters currently support holder-of-key token verification.
 Instead, keycloak adapters currently treat access and refresh tokens as bearer tokens.
 
+[[_proof-key-for-code-exchange]]
+*Proof Key for Code Exchange (PKCE)*
+
+PKCE prevents an attacker who steals an authorization code issued to a legitimate client from receiving the tokens in return to their stealing authorization code.
+
+The administrator can select the following three options:
+
+*Proof Key for Code Exchange Code Challenge Method*
+
+* (blank) : {project_name} does not apply PKCE unless the client sends PKCE's parameters appropriately to keycloak's authorization endpoint. It is default setting.
```

Comment:
I suggest:  It is the default setting.

Reply:
Yes, I'll revise the sentence as you suggested.

Reply:
Sorry, Takashi, but when I click View Changes, the message is that "it cannot find the content anywhere."  I am still learning how to view changes in GitHub.

Reply:
Okay, I found out where to see the changes.  Looks good. Thanks.